src/main/resources/application.properties

```diff
@@ -1,8 +1,21 @@
 spring.application.name=demo
 spring.custom.organization.name = amplimindcc
-spring.docker.compose.lifecycle-management=start_only
 
+
+##Enable for development
 spring.datasource.url=jdbc:postgresql://localhost:5432/postgres_db
+springdoc.swagger-ui.enabled=true
+spring.docker.compose.lifecycle-management=start_only
+
```

Comment:
Enable for development: need to be enabled for IDE run
Enable for image build: need to be enabled for creating a working image

Only one set shall be enabled at a time